bindings/java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
+networkTimeout=10000
```

Comment:
I'm guessing this will help the intermittent CI failures? If so, great 👍

Reply:
Don't know actually, It's just the default generated when I ran `gradle wrapper --gradle-version 8.0.2`